src/hotspot/share/opto/macro.cpp

```diff
@@ -41,6 +41,7 @@
 #include "opto/loopnode.hpp"
 #include "opto/macro.hpp"
 #include "opto/memnode.hpp"
+#include "opto/movenode.hpp"
```

Comment:
Unnecessary change?

Reply:
Oops, I forgot to remove the include from the old macro expansion logic. Thanks for the catch!